ote/src/cljs/ote/app/controller/route.cljs

```diff
@@ -57,27 +63,98 @@
                      %)
                   stops))))
 
+(defn- update-stop-times
+  "Copy departure and arrival time for stops from first trip. Stops can't have departure time in db, but in
+  ui they can."
+  [stops trips]
+  (let [departured-stops (map-indexed
+                           (fn [idx item]
+                             (assoc item ::transit/departure-time
+                                         (get-in (first trips) [::transit/stop-times idx ::transit/departure-time])))
+                           stops)
+        updated-stops (map-indexed
```

Comment:
can we do both updates with the same `map-indexed` call, just assoc both departure-time and arrival-time there